packages/design-system/src/components/tree.tsx

```diff
@@ -465,6 +474,7 @@ export const TreeNode = ({
       {...nodeProps}
       css={{ [treeNodeLevel]: level }}
       onKeyDown={handleKeydown}
+      isActionVisible={isActionVisible}
```

Comment:
You can set variable with css like above

Reply:
done